Documentation/op-guide/runtime-configuration.md

```diff
@@ -126,7 +126,7 @@ If adding multiple members the best practice is to configure a single member at
 #### Add a new member as learner
 
 Starting from v3.4, etcd supports adding a new member as learner / non-voting member.
-The motivation and design can be found in [design doc](https://etcd.readthedocs.io/en/latest/server-learner.html).
+The motivation and design can be found in [design doc](https://etcd.io/docs/v3.3.12/learning/learner/#raft-learner).
```

Comment:
This document is part of the repo, the link should be relative to the repo vs a hardcoded version. See examples here: https://raw.githubusercontent.com/etcd-io/etcd/master/Documentation/op-guide/clustering.md

Reply:
@hexfusion thanks, that sounds good. In the related issue https://github.com/etcd-io/etcd/issues/10989 we had a brief discussion and I think based on that @retroflexer used the etcd.io one. @retroflexer please use the other link in the format suggested by @hexfusion . Thanks!